src/components/Amount.vue

```diff
@@ -1,25 +1,38 @@
 <template>
     <span class="amount" :class="{ approx: showApprox && isApprox }">
         {{ formattedAmount }}
-        <span class="nim">NIM</span>
+        <span class="currency" :class="currency">{{currency}}</span>
     </span>
```

Comment:
How are you going to handle that in American English the currency should be displayed _in front_ of the value?

Reply:
As this was, and still is, used exclusively to display values for NIM, ETH and BTC, I went with what was needed, and not with everything imaginable.
There is the `CurrencyInfo` class in nimiq-utils, which could be used here to achieve that behaviour. As of now the component is not used in that capacity and `FiatAmount` does exist, too.

To also respond to your comment above, I think it makes sense to incorporate this functionality in one component as it basically only replaces a constant with a variable. The component itself does not get a lot more complex by doing that. However introducing the fiat part too is maybe not needed, as there already is `FiatAmount`, which also does have different internal workings.